testsuite/CMakeLists.txt

```diff
@@ -5,4 +5,8 @@ add_subdirectory(tabulated)
 add_subdirectory(system_test)
 add_subdirectory(minimize_energy)
 add_subdirectory(Array4DTest)
-add_subdirectory(fixed_list_types_test)
\ No newline at end of file
+add_subdirectory(fixed_list_types_test)
+add_subdirectory(RDFatomistic)
+add_subdirectory(AdressDensity)
+add_subdirectory(TDforce)
+add_subdirectory(FreeEnergyCompensation)
```

Comment:
I would move the 4 tests to `AdResS/TDforce` etc. and add a `CMakeLists.txt` in the `AdResS` folder and do `add_subdirectory(AdResS)` here.